src/s2/util/gtl/btree.h

```diff
@@ -73,6 +73,14 @@
 #include "s2/third_party/absl/strings/string_view.h"
 #include "s2/util/gtl/subtle/compressed_tuple.h"
 
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
Could you change size_type to std::make_signed<size_t>::type instead?

I'd like to keep any platform-specific code to a minimum and confine to s2/base if possible.

Reply:
Will check.